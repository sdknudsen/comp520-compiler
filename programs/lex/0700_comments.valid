
// Testing weird characters in comments!
// äåãøłúüö«áœéíóõ¡ºª¢€ħðþ‘’–×ÄÅØ~ĐŁÚÜÖÁ~ÑÉÆŒ~~
/* äåãøłúüö«áœéíóõ¡ºª¢€ħðþ‘’–×ÄÅØ~ĐŁÚÜÖÁ~ÑÉÆŒ~~ */ 1;

// if
// then
// else

