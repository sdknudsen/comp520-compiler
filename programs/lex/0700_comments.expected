INT(1)
SEMICOLON
EOF
